doc/nrf/releases/release-notes-latest.rst

```diff
@@ -73,6 +73,7 @@ Thread
 
 * Optimized ROM and RAM used by Thread samples.
 * Disabled Hardware Flow Control on the serial port in :ref:`coap_client_sample` and :ref:`coap_server_sample` samples.
+* Development support for nRF21540 FEM (front-end module) in the Thread samples.
```

Comment:
Please move up to line 73 and indent - this is not an update, but something that has been added.